ambry-messageformat/src/test/java/com.github.ambry.messageformat/BlobPropertiesTest.java

```diff
@@ -41,6 +43,17 @@
 public class BlobPropertiesTest {
 
   private final short version;
+  private static short serDeVersionSaved;
+
+  @BeforeClass
```

Comment:
do you want this to run before every test in the class or just once before any test of this class is run? If it is the former (which is what makes sense to me) - this has to be a `@Before`- not a `@BeforeClass`. Same thing with the `@AfterClass` and all tests

Reply:
`@BeforeClass` is what I wanted because the saving of version has to happen just once. As for `@After` vs. `@AfterClass`, this was really supposed to guard against a particular class not screwing up tests in other classes and that is why I chose `@AfterClass`. I suppose there is value in resetting after every test though so that unless explicitly asked for, every test tests the default version. Have changed `@AfterClass` to `@After`.